src/models/JensModel.py

```diff
@@ -53,6 +53,10 @@ def __init__(self, **kwargs):
 
         # create model
         self.model = self._create_model(kwargs["n_features"], kwargs["n_outputs"])
+        # Refactoring idea:
+        # n_features of a neuronal net is the number of inputs, so in reality n_features = window_size * n_features
+        # we could have another name for that
```

Comment:
No, n_features is the number of sensor features. I'm not sure what this refactoring would improve.